candi/bashible/common-steps/node-group/050_configure_and_start_containerd.sh.tpl

```diff
@@ -58,7 +60,7 @@ oom_score = 0
   gid = 0
   level = ""
 [metrics]
-  address = ""
+  address = "${discovered_node_ip}:1338"
```

Comment:
Without any authentication whatsoever?

Reply:
Containerd cannot support any authentication on metrics endpoint.

Reply:
Of course. But what if we deploy an intermediate kube-rbac-proxy and bind containerd metrics to a locahost?